src/tests/runTest.ts

```diff
@@ -18,13 +18,34 @@
 // All tests ran here are ones that require the vscode API
 import * as path from 'path'
 import * as cp from 'child_process'
+import { parseArgs } from 'util'
 import {
   runTests,
   resolveCliArgsFromVSCodeExecutablePath,
   downloadAndUnzipVSCode,
 } from '@vscode/test-electron'
 
 async function main() {
+  const {
+    values: { disable_cert_verification },
+  } = parseArgs({
+    options: {
+      disable_cert_verification: {
+        type: 'boolean',
+        short: 'k',
+        default: false,
+      },
+    },
+  })
+  if (disable_cert_verification) {
+    process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0'
+  }
+
+  // The version of VS Code to use for running the test suite in
+  const testVsCodeVersion = process.env.DAFFODIL_TEST_VSCODE_VERSION
+    ? process.env.DAFFODIL_TEST_VSCODE_VERSION
+    : 'stable'
```

Comment:
I noticed package.json defines vscode engine at 1.55, should we be testing with that same version unless overridden by the env variable?

Reply:
That said, I've never run the test suite against version 1.55.0, and if we're claiming that 1.55.0 is the minimum version we support, then perhaps we pin CI testing there.

Reply:
Note that the yarn.lock file pins vscode to 1.71.0, so that's probably what we actually test with. I'm not familiar enough with VS Code to know what a reasonable minimum target it.

Reply:
1.55.0 failed the tests, probably because the test suite wasn't in place at that time.  I upped the minimum version to 1.60.2, released October 4th, 2021 as this version works with the current test cases and has support for passing messages with array buffers that we need for the upcoming Data Editor.  I also made that the default for testing against rather than stable.